updated box to warning and shortened (#139)
aspnet/AspNetDocs: aspnet/includes/signalr/signalr-version-disambiguation.md

> [!WARNING]

> > This documentation is not for the latest version of SignalR. Take a look at [ASP.NET Core SignalR](/aspnet/core/signalr/introduction).

Reviewer: ```suggestion
> This documentation isn't for the latest version of SignalR. Take a look at [ASP.NET Core SignalR](/aspnet/core/signalr/introduction).
```

> > This documentation is not for the latest version of SignalR. Take a look at [ASP.NET Core SignalR](/aspnet/core/signalr/introduction) to keep up with the latest updates. 

Reviewer 2: "latest updates" makes it sound like Asp.Net Core SignalR is an update of Asp.Net SignalR
